File: Programmers/Lv1/Lv1_가운데_글자_가져오기.py
def solution(s):
    answer = ''
    s_len = len(s)
    if s_len in [1, 2]:
        return s

    if s_len % 2 == 0:
        return s[s_len // 2 - 1:s_len // 2 + 1]
    else:
        return s[s_len // 2]

File: JavaScript/Lv1_가운데_글자_가져오기.js
function solution(s) {
  const sLen = s.length;
  if (sLen === 1) return s;
  else if (sLen % 2 === 1) return s[(sLen - 1) / 2];
  else return s.slice((sLen - 2) / 2, sLen / 2 + 1);
}
